examples/authorize-beta/src/main/java/com/dropbox/core/examples/authorize_beta/IncrementalScopeAuthorize.java

```diff
@@ -0,0 +1,62 @@
+package src.main.java.com.dropbox.core.examples.authorize_beta;
+
+import com.dropbox.core.DbxAppInfo;
+import com.dropbox.core.DbxAuthFinish;
+import com.dropbox.core.DbxException;
+import com.dropbox.core.DbxRequestConfig;
+import com.dropbox.core.DbxWebAuth;
+import com.dropbox.core.IncludeGrantedScopes;
+import com.dropbox.core.TokenAccessType;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * Use include_granted_scopes when you ask for a small amount of permission at beginning, and
+ * then incrementally ask for more permission when user hit certain feature.
+ * The token contains all previously granted scopes.
+ */
+public class IncrementalScopeAuthorize {
+    public DbxAuthFinish authorize(DbxAppInfo appInfo) throws IOException {
+        // Run through Dropbox API authorization process
+        DbxRequestConfig requestConfig = new DbxRequestConfig("examples-authorize");
+        DbxWebAuth webAuth = new DbxWebAuth(requestConfig, appInfo);
+
+        // Ask for account_info.read scope
+        DbxWebAuth.Request webAuthRequest =  DbxWebAuth.newRequestBuilder()
+            .withNoRedirect()
+            .withTokenAccessType(TokenAccessType.OFFLINE)
+            .withScope("account_info.read")
+            .withIncludeGrantedScopes(IncludeGrantedScopes.USER)
```

Comment:
First call cannot include granted scopes

Reply:
Yeah I deleted this example. Actually I tried this example, if the user never connected the app before. The first call with include_granted_scopes won't throw error.